sql/2020/08_Accessibility/alt_text_length.sql

```diff
@@ -0,0 +1,28 @@
+#standardSQL
+# Alt text length
+SELECT
+  client,
+  MAX(alt_length) as max_alt_length,
+
+  percentile,
+  APPROX_QUANTILES(alt_length, 1000)[SAFE_ORDINAL(percentile * 10)] AS alt_length
```

Comment:
This should be compatible with the 100th percentile instead of `MAX` 
```suggestion
  APPROX_QUANTILES(alt_length, 1000)[OFFSET(percentile * 10)] AS alt_length
```